<launch>
    <arg name="node_name" default="camera1_control"/>
    <arg name="topic_name" default="compressed_camera1_images"/>
    <arg name="desired_fps" default="100"/>
    <arg name="usb_port" default="1"/>
    <node pkg="camera_control" type="camera_control.py" name="$(arg node_name)"
    args="$(arg node_name) $(arg topic_name) $(arg desired_fps) $(arg usb_port)" 
    required="true" output="screen"/>
</launch>